app/controllers/check_ins_controller.rb

```diff
@@ -0,0 +1,31 @@
+class CheckInsController < ApplicationController
+  # before_action :setup_variables, only: [:new, :create]
+  def index
+    @check_in = CheckInDecorator.new(check_in_params)
```

Comment:
Decorators take a Model when you create them. Also `def index` is for the `index` route which lists *all* check_ins, something we are not worrying about yet; so feel free to delete this entire method.